scripts/download-wp-server-files.ts

```diff
@@ -15,7 +15,7 @@ const FILES_TO_DOWNLOAD = [
 	{
 		name: 'sqlite',
 		description: 'SQLite files',
-		url: 'https://codeload.github.com/WordPress/sqlite-database-integration/zip/refs/heads/main',
+		url: 'https://downloads.wordpress.org/plugin/sqlite-database-integration.zip',
```

Comment:
Because the `sqlite-database-integration` GitHub [repository](https://github.com/WordPress/sqlite-database-integration/releases) names each release ZIP with the version number — e.g., `v2.1.10.zip` — [linking to the latest release](https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases#linking-to-the-latest-release) on GitHub does not work, as the ZIP file will always have a different name.

Reply:
Agreed. I originally split the source due to a "need" to fetch versions, but, as you noted, that is not actually necessary. With the latest changes, the wp.org source is now used in all locations.

Reply:
Thank you for making the changes 🙌